cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -507,6 +507,10 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 		a.AutoscalingContext.DebuggingSnapshotter.SetClusterNodes(l)
 	}
 
+	if err := a.processors.ProvisioningRequestProcessor.Process(nil); err != nil {
```

Comment:
I would make the `ProvisioningRequestProcessor.Process(nil)` take no arguments.
Only the sub-processors could receive the provreqs, which would be fetched by the combined processor.

Reply:
Done